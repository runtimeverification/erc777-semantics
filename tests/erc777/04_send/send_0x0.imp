test.preamble();

// send to 0x0 - must revert

test.send(0, 1, "");
