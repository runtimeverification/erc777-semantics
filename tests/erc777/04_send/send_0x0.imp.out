7 is sending 1 to 0 (data:)
REVERT
Balance of 7 is 70
Balance of 0 is 0
